doc/source/data_load/spark.md

```diff
@@ -1 +1,210 @@
+  .option("dbtable", table)
+  .option("user", user)
+  .option("password", pwd)
+  .option("batchsize", 100)
+  .option("isolationLevel", "NONE")
+  .option("checkpointLocation", checkpointLocation)
+  .start()
+
+query.awaitTermination()
+```
+
+## 2. Holohub Connector（write)
+
+### Jar Usage
+
+1.Emr Spark Sample
+
+```
+spark-shell --jars emr-datahub_shaded_2.11-2.0.0-SNAPSHOT.jar --driver-class-path emr-datahub_shaded_2.11-2.0.0-SNAPSHOT.jar
+```
+
+2.Spark Sample--Community Edition
+
+```
+spark-shell --jars emr-datahub_shaded_2.11-2.0.0-SNAPSHOT.jar
+```
+
+### Configuration Options
+
+```
+df.write
+  .format("datahub")
+  .option("access.key.id", "yourAccessKeyId")
+  .option("access.key.secret", "yourAccessKeySecret")
+  .option("endpoint", "datahubEndpoint")
+  .option("project", "yourDatabase")
+  .option("topic, "yourTable")
+  .option("batch.size", "100")
+  .save()
+```
+
+Highlight：
+
+• batch.size：In order to improve writing efficiency of micro-batch processing, by deafult, value 1, represents no micro-batch processing, support streaming mode. For batch processing it shall be replaced with other suitable values. 
+
+• format: Configured as datahub, write through holo instance by holohub interface.
+
+and decimal precision set up
+
+```
+df.option("decimal.precision", 38)
+  .option("decimal.scale", 18)
+```
+
+### Sample
+
+Create one test table in hologres
+
+```
+BEGIN;
+CREATE TABLE tb001 (
+  product_id BIGINT,
+  product_name TEXT,
+  price NUMERIC(38, 18),
+  out_of_stock bool,
+  weight double precision,
+  ts timestamptz
+);
+COMMIT;
+```
+
+Scala Code: Execute below command in terminal to open spark interactive shell where you can write interactive spark code for result-observing
+
+```
+spark-shell --jars emr-datahub_shaded_2.11-2.0.0-SNAPSHOT.jar
+```
+
+And then execute below scala code in spark-shell
+
+```
+val data = Seq(
+      Row(1L, "iphone", 1234.56, false, 123.45, "2020-05-21 00:00:00"),
+      Row(2L, "Huawei", 1234.56, true, 123.45, "2020-05-21 00:00:00")
+)
+
+// supported data types
+val schema = StructType(Array(
+  StructField("product_id", LongType),
+  StructField("product_name", StringType),
+  StructField("price", DecimalType),
+  StructField("out_of_stock", BooleanType),
+  StructField("weight", DoubleType),
+  StructField("ts", TimestampType)
+))
+
+val df = spark.createDataFrame(
+  spark.sparkContext.parallelize(data),
+  schema
+)
+
+df.write
+  .format("datahub")
+  .option(DatahubSourceProvider.OPTION_KEY_ACCESS_ID, 'my_accessKeyId')
+  .option(DatahubSourceProvider.OPTION_KEY_ACCESS_KEY, 'my_accessKeySecret')
+  .option(DatahubSourceProvider.OPTION_KEY_ENDPOINT, 'hgxxxx.xxxx.xxx:80')
+  .option(DatahubSourceProvider.OPTION_KEY_PROJECT, 'testdb')
+  .option(DatahubSourceProvider.OPTION_KEY_TOPIC, 'tb001)
+  .option("decimal.precision", 38)
+  .option("decimal.scale", 18)
+  .option("batch.size", "100")
+  .save()
+```
+
+## Data Types Mapping
+
+| Spark | Hologres |
```

Comment:
spark connector only support 6 primitive types? Flink connector supports 11 primitive types and array types

Reply:
While connect from holohub connector it only support these 6 primitive types. but while connective with JDBC it will shows the users one link where shows all the data type that hologres supports